### ISSUE TITLE
use ucontext_t instead of struct ucontext for platform-linuxarm.h

### ISSUE THREAD

**Issue by panthus** [Clozure/ccl#142]
The build fails for armv7h on archlinux with error:
```
./../platform-linuxarm.h:31:41: error: dereferencing pointer to incomplete type ‘ExceptionInformation {aka struct ucontext}’
```
I tried applying the patch from #80 (file platform-linuxx8632.h) to platform-linuxarm.h and that seems to fix it.

**Comment by xrme** [comment]
Is this on master or on the 1.11 branch? If it's master, I merged #140 which is intended to fix that.

If you're talking about 1.11, I haven't yet merged the fix there, but will very soon.

**Comment by xrme** [comment]
In fact, I merged a fix to 1.11 just now.

If you pull and rebuild to verify that the fix works, please close this issue.  Thanks for reporting the issue.

**Comment by panthus** [comment]
It was on 1.11, and yes that fixes it. Thanks!
